src/audio/iir.c

```diff
@@ -138,22 +144,10 @@ void iir_init_delay_df2t(struct iir_state_df2t *iir, int64_t **delay)
 {
 	iir->delay = *delay; /* Delay line of this IIR */
 	*delay += 2 * iir->biquads; /* Point to next IIR delay line start */
```

Comment:
what is this 2? channel number or size?

Reply:
The DF2 biquad contains two memory elements. There's the ASCII drawing at line 45. But better to comment the magic numbers.